smartsim/_core/_cli/build.py

```diff
@@ -367,6 +355,11 @@ def install_torch(self, device="cpu"):
 
     def check_onnx_install(self):
         """Check Python environment for ONNX installation"""
+        if not self.versions.ONNX:
+            raise SetupError(
+                "An onnx wheel is not available for this version of python for the "
```

Comment:
It would be friendlier to print the requested Python version here. It would be friendlier yet to state which versions of Python we _can_ support ONNX with.

Reply:
Good call! I changed the wording to include the current version of python they are using and a brief summary of what combinations of SS and RAI are supported based on their current platform.